### PR TITLE
don't specify ignorecase when calling isEquals and isEqualOrParent

### PR DESCRIPTION
**Pull request by aeschli**

### CODE REVIEW

**Comment by aeschli** [comment]
@bpasero @sandy081 Please have a quick look as I changes some your of your classes.

File: src/vs/workbench/contrib/files/browser/editors/fileEditorTracker.ts [hunk 1]
```diff
@@ -244,7 +243,7 @@ export class FileEditorTracker extends Disposable implements IWorkbenchContribut
 		});
 	}
 
-	private getIndexOfPath(path: string, candidate: string): number {
+	private getIndexOfPath(path: string, candidate: string, ignoreCase: boolean): number {
```
Comment (bpasero) [inline src/vs/workbench/contrib/files/browser/editors/fileEditorTracker.ts]: @aeschli I would just remove this parameter `ignoreCase: boolean` and inline it to `resources.hasToIgnoreCase(resource)`?

**Review by bpasero — APPROVED** [review]
@aeschli one code change suggested.

Also: should we simply not allow to set `ignoreCase` on `isEqual` and `isEqualOrParent`? Being able to overrule this decision is only asking for trouble imho.

You also might want to do a quick scan of usages of `isLinux` to see if there are more places where we make these assumptions but where we are not using your `resource` helpers.

**Review by sandy081 — APPROVED** [review]

**Comment by aeschli** [comment]
I reverted the change of the default of ignorecase for remote resources.
The PR just fixes all callers to not define ignorecase.

**Comment by bpasero** [comment]
@aeschli possibly this one ([`explorerModel.ts`](https://github.com/Microsoft/vscode/blob/ef1de00/src/vs/workbench/contrib/files/common/explorerModel.ts#L337)) should get the same treatment as well.

**Comment by aeschli** [comment]
@bpasero Thanks, added 614db8bc770b2a989ce01f76946e473b31f79922

**Pull request merged**